### DISCUSSION TITLE
Can deep potential be used in other packages?

### DISCUSSION THREAD

**Discussion by Zhang-pchao** [deepmodeling/deepmd-kit#551] [Q&A]
We can use deep potential (.pd) with LAMMPS, i-PI and ASE, is it possible to use .pd for some popular first-principles software in the future?

**Comment by amcadmus** [comment]
Use a Deep Potential model in python is straight forward, please check the example
https://github.com/deepmodeling/deepmd-kit/blob/devel/doc/use-deepmd-kit.md#model-inference

In other C/C++/FORTRAN written packages, one may call the C++ API of DeePMD-kit. Please check 
```
$deepmd_source_dir/source/api_cc
```

**Reply by Zhang-pchao** [reply]
Thank you for your reply!

Is it possible to use frozen_model.pb with Gaussian in the future?
